CTP/shell/src/com/navercorp/cubridqa/shell/result/FeedbackFile.java

```diff
@@ -84,30 +98,76 @@ public void onTaskContinueEvent() {
 
 	@Override
 	public void onTaskStopEvent() {
+		showTestResult();
+		
 		long taskStopTime = System.currentTimeMillis();
 		println("[TEST STOP] Current Time is " + new Date(), "Elapse Time:" + ((taskStopTime - this.taskStartTime)));
 		feedbackLog.close();
 	}
 
 	@Override
 	public void setTotalTestCase(int tbdNum, int macroSkippedNum, int tempSkippedNum) {
+		this.totalCaseNum = tbdNum;
```

Comment:
if(is NOT continue mode) {
   save status data
} 

In addition, in onTaskContinueEvent(), when continue mode, load status data from file.